src/test/common/steps/pages.js

```diff
@@ -64,6 +66,7 @@ class Pages {
     this.serverError = new ServerError(this.driver)
     this.pageNotFound = new PageNotFound(this.driver)
     this.privacyNotice = new PrivacyNotice(this.driver)
+    this.unsuccessfulApplication = new UnsuccessfulApplication(this.driver)
     // NOTE: This map should contain all page objects, and not the Generic Page as this doesn't itself represent a page
     this.allPages = [this.overview, this.enterName, this.enterNino, this.enterDOB, this.areYouPregnant, this.cardAddress,
```

Comment:
You need to add `unsuccessfulApplication` to the `allPages` array.

Reply:
As discussed, not needed.